wp-content/themes/core/pcss/utilities/variables/_colors.pcss

```diff
@@ -3,14 +3,25 @@
 	 * Primary Palette
 	 * ----------------------------------------------------------------------------- */
 
-	--color-primary: #21a6cb;
-	--color-secondary: darken(var(--color-primary), 0.08);
+	--color-java: #1ca8c7;
+	--color-pickled-bluewood: #2b474f;
+	--color-matisse: #1a7f9d;
+	--color-gossamer: #078e87;
+	--color-mandy: #e4554a;
+	--color-casablanca: #f4af49;
+	--color-saffron-mango: #f8ce4d;
+
+	/* Global Color Vars */
+	--color-primary: var(--color-java);
+	--color-secondary: var(--color-pickled-bluewood);
+	--color-hover: var(--color-matisse);
```

Comment:
Given we already have `color-link-hover` below, I'd argue this is redundant and we should nix it in favor of that or vice versa

Reply:
Agree, nixed in favor of `color-link-hover`